--- src/Carbon/Doctrine/DateTimeImmutableType.php	2023-12-10 16:33:53
+++ src/Carbon/Doctrine/DateTimeImmutableType.php	2024-02-09 11:36:50
@@ -17,7 +17,7 @@
     /**
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
-    public function convertToPHPValue(mixed $value, AbstractPlatform $platform): ?DateTimeImmutable
+    public function convertToPHPValue(mixed $value, AbstractPlatform $platform): ?CarbonImmutable
     {
         return $this->doConvertToPHPValue($value);
     }
