--- src/Mapping/JoinColumn.php	2024-02-03 17:50:09
+++ src/Mapping/JoinColumn.php	2024-02-08 14:22:27
@@ -5,7 +5,13 @@
 namespace Doctrine\ORM\Mapping;

 use Attribute;
+use Doctrine\Common\Annotations\Annotation\NamedArgumentConstructor;

+/**
+ * @Annotation
+ * @NamedArgumentConstructor()
+ * @Target({"PROPERTY","ANNOTATION"})
+ */
 #[Attribute(Attribute::TARGET_PROPERTY | Attribute::IS_REPEATABLE)]
 final class JoinColumn implements MappingAttribute
 {
